src/main/java/com/coveros/selenified/utilities/Listener.java

```diff
@@ -160,11 +160,11 @@ private void recordResult(ITestResult result) {
         }
         // update our reporter logger
         String testName = getTestName(result);
-        Browser browser = (Browser) result.getAttribute(BROWSER_INPUT);
+        Browser browser = (Browser) result.getAttribute(BROWSER);
         if (browser != null) {
             Reporter.log(
-                    Result.values()[result.getStatus()] + OUTPUT_BREAK + browser.getName() + OUTPUT_BREAK + LINK_START +
-                            getFolderName(result) + "/" + testName + browser.getName() + LINK_MIDDLE + testName +
+                    Result.values()[result.getStatus()] + OUTPUT_BREAK + browser.getDetails() + OUTPUT_BREAK + LINK_START +
+                            getFolderName(result) + "/" + outputFile.getFileName() + LINK_MIDDLE + testName +
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "outputFile" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS2259)